src/game/engine/target.cpp

```diff
@@ -101,6 +101,11 @@ target_t As_Target(CellClass *cell)
     return cell != nullptr ? As_Target(cell->Cell_Number()) : 0;
 }
 
+target_t As_Target(AbstractClass *object)
```

Comment:
This isn't required, I already added a method to AbstractClass to return itself as a target_t.